website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -0,0 +1,240 @@
+---
+layout: docs
+page_title: Upgrades
+description: >-
+  This topic describes how to upgrade Consul API Gateway.
+---
+
+# Upgrades
+
+This topic describes how to upgrade Consul API Gateway.
+
+## Breaking Changes
+
+Consul API Gateway v0.2.0 introduces a breaking change for people upgrading from Consul API Gateway v0.1.0. Routes with a `backendRef` defined in a different namespace now require a [`ReferencePolicy`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy) that explicitly allows traffic from the route's namespace to the `backendRef`'s namespace. 
+
+## Requirements
+
+Ensure that the following requirements are met prior to upgrading:
+
+- Consul API Gateway should be running version v0.1.0. 
+- You should have the ability to run `kubectl` CLI commands.
+- `kubectl` should be configured to point to the cluster containing the installation you are upgrading.
+- You should have the following rights on your Kubernetes cluster:
+  * `HTTPRoute.read` 
+  * `TCPRoute.read` 
+  * `ReferencePolicy.create` 
+- (Optional) The [jq](https://stedolan.github.io/jq/download/) command line processor for JSON can be installed, which will ease route retrieval during the upgrade process. 
+
+## Procedure
+
+1. Verify the current version of the `consul-api-gateway-controller` `Deployment`:
+
+  ```shell-session
+  $ kubectl get deployment --namespace consul consul-api-gateway-controller --output=jsonpath= "{@.spec.template.spec.containers[?(@.name=='api-gateway-controller')].image}"
+  ```
+
+  You should receive the following response:
+
+  ```log
+  "hashicorp/consul-api-gateway:0.1.0"
+  ```
+
+1. Retrieve all routes that have a backend in a different namespace. If you have installed the `jq` utility, you can skip to [step 4](#jq-command). Otherwise, issue the following command to get all `HTTPRoutes` and `TCPRoutes` across all namespaces: 
```

Comment:
```suggestion
1. Retrieve all routes that have a backend in a different namespace. If you have installed the [`jq`](https://stedolan.github.io/jq/) utility, you can skip to [step 4](#jq-command). Otherwise, issue the following command to get all `HTTPRoutes` and `TCPRoutes` across all namespaces: 
```
Just in case someone is not familiar with jq.